server/game/cards/01_SOR/units/FirstLegionSnowtrooper.ts

```diff
@@ -0,0 +1,22 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { NonLeaderUnitCard } from '../../../core/card/NonLeaderUnitCard';
+import { KeywordName } from '../../../core/Constants';
+import * as EnumHelpers from '../../../core/utils/EnumHelpers';
+export default class FirstLegionSnowtrooper extends NonLeaderUnitCard {
+    protected override getImplementationId() {
+        return {
+            id: '4619930426',
+            internalName: 'first-legion-snowtrooper',
+        };
+    }
+
+    public override setupCardAbilities() {
+        this.addConstantAbility({
+            title: 'While attacking a damaged unit, this unit gets +2/+0 and gains Overwhelm.',
+            condition: (context) => context.source.isAttacking() && context.source.activeAttack?.target.isUnit() && (EnumHelpers.isArena(context.source.activeAttack?.target.location) && context.source.activeAttack?.target.damage > 0),
```

Comment:
Sorry, one more request to make here - this line is super long, do you mind breaking it up into multiple lines

Reply:
sure no problem!

Reply:
broke into three lines.